_sub/compute/k8s-traefik-flux/dependencies.tf

```diff
@@ -62,7 +62,7 @@ locals {
     "apiVersion" = "kustomize.config.k8s.io/v1beta1"
     "kind"       = "Kustomization"
     "resources" = [
-      "https://github.com/dfds/platform-apps/apps/${var.deploy_name}"
+      "${var.gitops_apps_repo_url}/apps/${var.deploy_name}?ref=${var.gitops_apps_repo_branch}"
```

Comment:
Shouldn't you run this through `urlencode`?

Reply:
Does a branch with a forward slash work?

Reply:
Yes, it works, it has been tested in my sandbox. Here is an output from my sandbox:

`platform-apps-sources                     	feature/pss-psa@sha1:9d3ee061	False    	True 	Applied revision: feature/pss-psa@sha1:9d3ee061`